File: Arrays/Python/Smallest_Positive_Missing_Number.py
class Solution:
    
    #Function to find the smallest positive number missing from the array.
    def missingNumber(self,arr,n):
        #Your code here
        arr.sort()
        s=1
        for i in range(0,len(arr)):
            if s==arr[i]:
                s+=1
        return s

File: Solutions/Java/Smallest_Positive_Missing_Number.java
class Solution
{
    //Function to find the smallest positive number missing from the array.
    static int missingNumber(int arr[], int size)
    {
        // Your code here
        Arrays.sort(arr);
        int s=1;
        for (int i=0; i<size; i++) {
            if (s==arr[i]) {
                s++;
            }
        }
        return s;
    }
}
